src/main/java/org/jboss/pull/player/Util.java

```diff
@@ -88,7 +90,20 @@ static boolean optionalBoolean(final String name, final boolean defaultValue) {
     private static class PropertiesHolder {
         static final Properties PROPERTIES = new Properties();
         static {
-            try (Reader reader = Files.newBufferedReader(BASE_DIR.toPath().resolve("player.properties"), StandardCharsets.UTF_8)) {
+            File[] children = BASE_DIR.listFiles((dri, name) -> name.endsWith("player.properties"));
```

Comment:
this seems a bit unclear, is your intent to have say, player.properties and then 1-player.properties, or am I misunderstanding the .endWith?

Reply:
Yes, that was the idea. A 'startsWith' could work too. Perhaps I overly value standard file extensions. ;)